arcgis-ios-sdk-samples/Utility Network/Display utility association/DisplayUtilityAssociationVC.swift

```diff
@@ -0,0 +1,141 @@
+    
+    private let utilityNetwork = AGSUtilityNetwork(url: URL(string: "https://sampleserver7.arcgisonline.com/arcgis/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer")!)
+    private let maxScale = 2000.0
+    private var associationsOverlay = AGSGraphicsOverlay()
+    private let attachmentSymbol = AGSSimpleLineSymbol(style: .dot, color: .green, width: 5)
+    private let connectivitySymbol = AGSSimpleLineSymbol(style: .dot, color: .red, width: 5)
+    
+    func loadUtilityNetwork() {
+        utilityNetwork.load { [weak self] error in
+            if let error = error {
+                self?.presentAlert(error: error)
+                return
+            } else {
+                guard let self = self else { return }
+                // Get all the edges and junctions in the network.
+                let edges = self.utilityNetwork.definition.networkSources.filter { $0.sourceType == .edge }
+                let junctions = self.utilityNetwork.definition.networkSources.filter { $0.sourceType == .junction }
+
+                // Add all edges that are not subnet lines to the map.
+                edges.filter { $0.sourceUsageType != .subnetLine }.forEach { source in
+                    self.mapView.map?.operationalLayers.add(AGSFeatureLayer(featureTable: source.featureTable))
+                }
+                // Add all the junctions to the map.
+                junctions.forEach { source in
+                    self.mapView.map?.operationalLayers.add(AGSFeatureLayer(featureTable: source.featureTable))
+                }
+                
+                // Create a renderer for the associations.
+                let attachmentValue = AGSUniqueValue(description: "Attachment", label: "", symbol: self.attachmentSymbol, values: [AGSUtilityAssociationType.attachment])
+                let connectivityValue = AGSUniqueValue(description: "Connectivity", label: "", symbol: self.connectivitySymbol, values: [AGSUtilityAssociationType.connectivity])
+                self.associationsOverlay.renderer = AGSUniqueValueRenderer(fieldNames: ["AssociationType"], uniqueValues: [attachmentValue, connectivityValue], defaultLabel: "", defaultSymbol: nil)
+                
+                // Populate the legened.
+                self.attachmentSymbol.createSwatch(withBackgroundColor: nil, screen: .main) { [weak self] (image, error) in
+                    if let error = error {
+                        print("Error creating swatch: \(error)")
+                    } else {
+                        self?.attachmentBBI.image = image
+                    }
+                }
+                self.connectivitySymbol.createSwatch(withBackgroundColor: nil, screen: .main) { [weak self] (image, error) in
+                    if let error = error {
+                        print("Error creating swatch: \(error)")
+                    } else {
+                        self?.connectivityBBI.image = image
```

Comment:
```suggestion
                        self?.connectivityBBI.image = image?.withRenderingMode(.alwaysOriginal)
```
This will render the image as-is, rather than with tint color applied to a templated `UIImage`.